github/data_source_github_organization_external_identities.go

```diff
@@ -0,0 +1,127 @@
+package github
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/shurcooL/githubv4"
+)
+
+func dataSourceGithubOrganizationExternalIdentities() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGithubOrganizationExternalIdentitiesRead,
+
+		Schema: map[string]*schema.Schema{
+			"identities": {
```

Comment:
Personally I think that you could drop the `identities` field entirely, and just expose `login`, `saml_identity` and `scim_identity` directly in the root level instead. Unless you have a reason for nesting it?

Reply:
Nevermind me. I forgot for a second that you return a list. This is the way it should be